doc/tech/approach.md

```diff
@@ -3,97 +3,226 @@
+
+The Overlay Networking separation is often done through encapsulation of Tenants' traffic using overlay protocols e.g. through VxLAN or EVPN over the Underlay Network e.g. based on L2 (VLAN) or L3 (IP) networks.
+
+In some instances, the SW Virtualisation Tenants can bypass the Overlay Networking encapsulation to achieve better performance or network visibility/control. A common method to bypass the Overlay Networking encapsulation normally done by the SW Virtualisation Layer, is the VNF/CNF usage of SR-IOV that effectively take over the Physical and Virtual Functions of the NIC directly into the VNF/CNF Tenant. In these cases, the Underlay Networking must handle the separation e.g. through a Virtual Termination End Point (VTEP) that encapsulate the Overlay Network traffic.
+
+> **Note:** Bypassing the Overlay Networking layer is a violation of the basic CNTT decoupling principles, but in some cases unavoidable with existing technologies and standards. Until suitable technologies and standards are developed, CNTT have a set of agreed exemptions that forces the Underlay Networking to handle the bypassed Overlay Networking separation.
+
+VTEP could be manually provisioned in the Underlay Networking or be automated and controlled through a Software Defined Networking controller interfaces into the underlying networking in the HW Infrastructure Layer. 
+
+<a name="5.1.3.2"></a>
+#### 5.1.3.2 Hardware and Software Infrastructure Layer concepts
+
+The Cloud Infrastructure can be considered to be composed of two distinct layers, referred to as HW Infrastructure Layer and SW Infrastructure Layer. When there are multiple separated simultaneously deployed SW Virtualisation domains, the architecture and deployed implementations must enable each of them to be in individual non-dependent administrative domains. The HW Infrastructure must also be enabled to be a fully separated administrative domain from all of the SW Virtualisation domains. 
+
+For Cloud implementations of multiple well separated simultaneous SW Virtualisation domains on a shared HW Infrastructure there must be a separation of the hardware resources e.g. servers and the Underlay Networking resources that interconnect the hardware resources e.g. through a switching fabric.
+
+To allow multiple separated simultaneous SW Virtualisation domains onto a shared switching fabric there is a need to split up the Underlay Networking resources into non overlapping addressing domains on suitable protocols e.g. VxLAN with their VNI Ranges. This separation must be done through an administrative domain that could not be compromised by any of the individual SW Virtualisation domains either by malicious or unintentional Underlay Network mapping or configuration.
```

Comment:
Switches provide a (mostly) ubiquitous forwarding plane, and generally aren't split up, per se. Maybe we can clarify this so it better reflects the natural usage models.

"...into non overlapping addressing domains on suitable protocols e.g. VxLAN..." - The addressing domains **_on_** the encapsulation protocols (e.g. VXLAN), _may_ or _may not_ be overlapping. For example, if 2 different tenants pick some private IPs out of popular private address space (e.g. 10.0.0.0/8 or 192.168.0.0/16), they may very well overlap, and this _**must**_ be supported by CNTT. It's the encapsulation protocol(s)' end-points themselves, that must have non-overlapping addresses. 

"...domain that could not be compromised..."  -> that **_cannot_** be compromised

Reply:
Updated file slightly based on the comments.

In the RM we cannot expect the Forwarding Plane to be implemented by switches. The implementation in the HW layer can be done by a single physical switch, a set of physical switches arranged in some interconnected way, in SmartNICs (managed by a HW Infrastructure manager) or any combinations of these. This must be the foundation for the RM. 

Agree with your statement that it is the encapsulation endpoints that must not be overlapping, when encapsulation is the method of separation. I also agree that CNTT must support SW Virtualization domains (or Bare Metal applications) to have  overlapping IP addresses, but I feel that is just one of the many things CNTT must ensure and those are more valid for requirement chapters. In this HW & SW Infrastructure concept chapter I have though clarified a bit that separation can be done through encapsulation and exemplified it with VxLAN and its VNI ranges allowing SW Virtualization IP address overlap without any risk for collisions. In reality it is often more complex than that and several other addresses must not overlap to avoid forwarding pipeline confusion.

Clarified that it "cannot be compromised" as per suggestion.